docs(fx112-css-overflow): Add note for 'overlay' and update other descriptions (#25749)
mdn/content: files/en-us/web/css/overflow-x/index.md

{{CSSRef}}

> The **`overflow-x`** [CSS](/en-US/docs/Web/CSS) property sets what shows when content overflows a block-level element's left and right edges. This may be nothing, a scroll bar, or the overflow content. It is one of the constituent properties of the shorthand [`overflow`](/en-US/docs/Web/CSS/overflow) property.

Reviewer: Nothing wrong with this, but if it were me, I'd simplify (same comment for overflow-y)

```suggestion
The **`overflow-x`** [CSS](/en-US/docs/Web/CSS) property sets what shows when content overflows a block-level element's left and right edges. This may be nothing, a scroll bar, or the overflow content. This property may also be set using the [`overflow`](/en-US/docs/Web/CSS/overflow) shorthand property.
```
